### ISSUE TITLE
Make tooltips keyboard accessible

### ISSUE THREAD

**Issue by hanshillen** [cgkineo/adapt-pageNav#33]
The pageNav button tooltips only appear on mouseover, which means they fail WCAG 2.1.1. 

Per [the APG Tooltip pattern](https://www.w3.org/WAI/ARIA/apg/patterns/tooltip/), the tooltips should also appear on focus, and disappear on blur or <kbd>esc</kbd>.

**Comment by oliverfoster** [comment]
Todo: Need to use the tooltip api in the fw now. And update that api with any relevant changes from this pr.

**Comment by oliverfoster** [comment]
Move to https://github.com/cgkineo/adapt-pageNav/pull/49
